scripts/lib/mkcloud-common.sh

```diff
@@ -533,6 +531,13 @@ function dist_to_image_name
     echo "$image.qcow2"
 }
 
+: ${zypper:=run_zypper}
+function run_zypper
+{
+    local params=${zypper_override_params:---non-interactive --gpg-auto-import-keys --no-gpg-checks}
+    $sudo zypper $params "$@"
+}
```

Comment:
Oh right. Maybe I assumed that it will work like `sshrun` where the functions of qa_crowbarsetup will be sourced.
Ok, will revert these lines.